package.json

```diff
@@ -22,8 +22,8 @@
     "@types/angular-animate": "^1.5.10",
     "@types/angular-mocks": "^1.7.0",
     "@types/ckeditor": "^4.9.2",
-    "@types/d3": "^5.7.2",
-    "@types/google.visualization": "0.0.48",
+    "@types/d3": "^3.5.40",
```

Comment:
Why are we downgrading?

Reply:
I think it did not get merged with #6969 and hence this and the other downgrades. These are unintentional.